From c26263c1aa36b475426dce3e6953b34efab1a696 Mon Sep 17 00:00:00 2001
From: Dimitar Dimitrov <dimitar@dinux.eu>
Date: Sun, 17 Mar 2019 22:08:01 +0200
Subject: [PATCH 16/22] testsuite: Mark PRU as having no alignment requirements

gcc/testsuite/ChangeLog:

2019-03-17  Dimitar Dimitrov  <dimitar@dinux.eu>

        * lib/target-supports.exp
	(check_effective_target_natural_alignment_32): Add pru-*-*.

Signed-off-by: Dimitar Dimitrov <dimitar@dinux.eu>
---
 gcc/testsuite/lib/target-supports.exp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/testsuite/lib/target-supports.exp b/gcc/testsuite/lib/target-supports.exp
index 01472879062..5a4aa585a4d 100644
--- a/gcc/testsuite/lib/target-supports.exp
+++ b/gcc/testsuite/lib/target-supports.exp
@@ -6287,7 +6287,7 @@ proc check_effective_target_natural_alignment_32 { } {
     # FIXME: 32bit powerpc: guaranteed only if MASK_ALIGN_NATURAL/POWER.
     return [check_cached_effective_target_indexed natural_alignment_32 {
       if { ([istarget *-*-darwin*] && [is-effective-target lp64])
-	    || [istarget avr-*-*] } {
+	    || [istarget avr-*-*] || [istarget pru-*-*] } {
 	   return 0
 	 } else {
 	   return 1
-- 
2.11.0

